gef-ui/server/controllers/exporters-address/index.js

```diff
@@ -25,8 +26,9 @@ const validateExportersAddress = async (req, res) => {
     const { params, body } = req;
     const { correspondence, postcode } = body;
     const { dealId } = params;
+    const { userToken } = req.session;
```

Comment:
I'll leave it up to you whether you want to destructure like the previous comment everywhere this pattern is present :)

Reply:
I'll leave as-is, thanks :)